gradle-baseline-java/src/test/groovy/com/palantir/baseline/BaselineIdeaIntegrationTest.groovy

```diff
@@ -86,6 +86,48 @@ class BaselineIdeaIntegrationTest extends AbstractPluginTest {
         rootIpr.contains('<component name="CopyrightManager" default="999_palantir.txt">')
     }
 
+    def 'Idea project has copyright configuration when importing'() {
+        when:
+        buildFile << standardBuildFile
+        System.setProperty("idea.active", "true")
```

Comment:
since you're setting this, you need an `@RestoreSystemProperties` on the method

see https://mrhaki.blogspot.com/2015/09/spocklight-undo-changes-in-java-system.html